metrics/CryptographyEncryptionKeyManagement/SecurityReaderRoleAssigned/SecurityReaderRoleAssigned.yml

```diff
@@ -0,0 +1,11 @@
+# ====== Metadata ======
+id: SecurityReaderRoleAssigned
+description: This rule assesses whether a [RoleAssignment] covering cryptographic oversight responsibilities has the property [p1:securityReaderRoleAssigned] indicating that at least one [Identity] is assigned to fulfil the responsibility.
```

Comment:
As far as I can see, the RoleAssignment Resource does not yet have any properties defined in the ontology. So, we need to adapt there and add something like a list of permissions.